_episodes/06-script.md

```diff
@@ -312,14 +312,17 @@ $ history | tail -n 5 > redo-figure-3.sh
 The file `redo-figure-3.sh` now contains:
 
 ~~~
-297 bash goostats -r NENE01729B.txt stats-NENE01729B.txt
+297 bash goostats NENE01729B.txt stats-NENE01729B.txt
 298 bash goodiff stats-NENE01729B.txt /data/validated/01729.txt > 01729-differences.txt
 299 cut -d ',' -f 2-3 01729-differences.txt > 01729-time-series.txt
 300 ygraph --format scatter --color bw --borders none 01729-time-series.txt figure-3.png
 301 history | tail -n 5 > redo-figure-3.sh
 ~~~
 {: .source}
 
+
+(`goostats` is a shell script which calculates some complicated statistics from a datafile -- the first argument -- and writes them to a file -- the second argument; the script `goodiff` compares two datafiles provided as arguments. Nelle's superviser provided them without too many explainations.)
```

Comment:
1. I suggest renaming datafile (not a word) to either "input file", "file" or "protein sample file".

2. I also suggest breaking up the first sentence into two, as `goostats` and `goodiff` are sufficiently different scripts. `goodiff` is not supposedly comparing files, but rather "Compar[ing] the statistics for each protein with corresponding statistics for each other protein ". I would re-word as "`goodiff` is a script for comparing statistics between two files, provided as arguments".

3. A few small typos: superviser->supervisor, explainations->explanations